dask_jobqueue/tests/test_oar.py

```diff
@@ -28,28 +28,55 @@ def test_header():
         assert "#OAR -q regular" in cluster.job_header
         assert "#OAR -t besteffort" in cluster.job_header
 
-    with OARCluster(cores=4, memory="8GB") as cluster:
+    with OARCluster(
+        cores=4, memory="8GB", oar_mem_core_property_name="memcore"
+    ) as cluster:
         assert "#OAR -n dask-worker" in cluster.job_header
         assert "walltime=" in cluster.job_header
+        assert "#OAR -p memcore" in cluster.job_header
         assert "#OAR --project" not in cluster.job_header
         assert "#OAR -q" not in cluster.job_header
 
+    with OARCluster(
+        walltime="00:02:00",
+        processes=4,
+        cores=8,
+        memory="28GB",
+        oar_mem_core_property_name="mem_core",
+    ) as cluster:
+        assert "#OAR -n dask-worker" in cluster.job_header
+        assert "#OAR -l /nodes=1/core=8,walltime=00:02:00" in cluster.job_header
+        assert "#OAR -p mem_core>=3337" in cluster.job_header
```

Comment:
Be careful with computation like this, we've already seen errors with memory string parsing. At least, I would suggest to sue `GiB` syntax in the `OARCluster` constructor, so this can be more easily divided into MiB.